MahApps.Metro/Controls/Helper/TextBoxHelper.cs

```diff
@@ -41,6 +41,14 @@ public class TextBoxHelper
 
         private static readonly DependencyProperty IsSpellCheckContextMenuEnabledProperty = DependencyProperty.RegisterAttached("IsSpellCheckContextMenuEnabled", typeof(bool), typeof(TextBoxHelper), new FrameworkPropertyMetadata(false, UseSpellCheckContextMenuChanged));
 
+        private static readonly Func<NumericUpDown, int> NumericUpDownTextLength = control => control.Value.HasValue ? 1 : 0;
+        private static readonly Func<PasswordBox, int> PasswordBoxTextLength = control => control.Password.Length;
+        private static readonly Func<TextBox, int> TextBoxTextLength = control => control.Text.Length;
+
+        private static readonly Func<NumericUpDown, Action> NumericUpDownFocus = control => control.SelectAll;
+        private static readonly Func<PasswordBox, Action> PasswordBoxFocus = control => control.SelectAll;
+        private static readonly Func<TextBox, Action> TextBoxFocus = control => control.SelectAll;
+
```

Comment:
@punker76 we could also inline this fields. But what do you think of both generic methods? Should simplify further development if we add *UseFloatingWatermark* for other controls.